.circleci/config.yml

```diff
@@ -149,7 +149,7 @@ jobs:
       - vault/get-secrets:
           template-preset: semantic-release
```

Comment:
Wouldn't it be better to change this to `packages-read` and keep the env name?
If we don't need to write, we should use the token with fewer privileges

Reply:
I think this would not work in this case, because we update the Changelog files and we push it back to github, so we need some tokens that don't exist on the `packages-read`, the `GIT_AUTHOR_EMAIL` and `GIT_AUTHOR_NAME`

Reply:
ah - i thought we only read. my bad